### DISCUSSION TITLE
anotação fixa em toda OS

### DISCUSSION THREAD

**Discussion by opj19** [RamonSilva20/mapos#1692] [Ideas]
tem como colocar uma frase fixa em todas as ordem de serviços que forem impressa 

tipo se o cliente deixar o equipamento na loja mais de 90 dias o produto vai ser descartado

**Comment by andremp15** [comment]
Oi amigo mas esse tipo de mensagem ou atitude é ilegal o que você pode estar fazendo é cobrar durante a estadia do equipamento na sua assistência e gerando valores assim o cliente não vai ter como pagar ou fica inviável para ele isso você pode fazer pelo código do consumidor

**Comment by SuperPorte** [comment]
Vai em:
application/views/os/adicionarOs.php
A partir da linha 86 Digite o texto que você precisa. Exemplo:

                                        <textarea class="span12 editor" name="descricaoProduto" id="descricaoProduto" cols="30" rows="5">(_)PC (_)Notebok </br>Marca: </br>Modelo: </br>SERIAL: </br></textarea>

O que você digitar ANTES de </textarea> vai ficar fixo na hora de adicionar a OS.
